server/app/repository/ApplicationStatusRepository.java

```diff
@@ -0,0 +1,41 @@
+package repository;
+
+import io.ebean.DB;
+import io.ebean.Database;
+import java.util.List;
+import java.util.Optional;
+import javax.inject.Inject;
+import models.ApplicationStatusesModel;
+import models.StatusLifecycleStage;
+
+/** A repository class used to interact with the application_statuses table. */
+public final class ApplicationStatusRepository {
```

Comment:
**suggestion**: Re my comment, naming this `ApplicationStatusSetRepository` helps clean up the naming pattern.